distributions.py

```diff
@@ -90,8 +90,11 @@ def sample_location(bounds):
  @param     mat a Material object that contains information about the material
  @return    An interaction type (0 = capture, 1 = fission)
 '''
-def sample_fission(mat):
-    return int(random.random() < mat.sigma_f / mat.sigma_a)
+def sample_fission(mat, g, new_g):
+
+    # I'm not sure if I'm using the right item in sigma_s
+    return int(random.random() < mat.sigma_f[g] \
+            / (mat.sigma_t[g]-mat.sigma_s[g][new_g]))
```

Comment:
The denominator should be ``mat.sigma_t[g]-  sum(mat.sigma_s[g])`` since the denominator should really be the absorption cross-section which is the total cross section minus all scattering from energy group ``g``. However perhaps a better way to do this (to avoid unnecessary computational work) is to create a new attribute in ``Material`` to be the absorption cross section ``sigma_a`` allowing the denominator to simply be ``mat.sigma_a[g]``. The ``sigma_a`` attribute could be computed when the material is initialized.